bin/cli.js

```diff
@@ -81,6 +99,11 @@ function rewrite(cb){
   walkdir(dir,function(p,stat){
     if(p.substr(p.length-3) !== '.js') return
     if(~p.indexOf('.git') || ~p.indexOf('node_modules')) return
+    for (var i = 0; i < ignorePaths.length; i++){
```

Comment:
Ah, true. I think that we could do that so long as we normalized the end of the inputted paths such that they matched whatever is returned as it's being walked.